src/main/java/duke/command/UnknownCommand.java

```diff
@@ -0,0 +1,41 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * This command is to be executed when the user enters
+ * an unknown command keyword.
+ */
+public class UnknownCommand extends Command {
+
+    /**
+     * Sole constructor of UnknownCommand.
+     */
+    public UnknownCommand() {
+
+    }
+
+    /**
+     * Throws a DukeException as the user has given an unknown command keyword.
+     * @param tasks the specified TaskList object.

```

Comment:
It would be better to leave a line between description and @param in the JavaDoc. I notice this problem in other methods as well.

Reply:
Thanks for the feedback! Resolved in 3d0220ab8b77dafa39a49a690592363485d70a1f.